### PR TITLE
Handle Markdown and Jupyter Notebooks

### PR DESCRIPTION
**Pull request by o2sh**
Fix for #180 

New code distribution of the repo shown in the issue:

lacmus

### CODE REVIEW

File: src/onefetch/language.rs [hunk 1]
```diff
@@ -163,18 +163,36 @@ impl Language {
...
+            if code == 0 {
+                continue;
+            }
+
+            stats.insert(Language::from(*language_type), code as f64);
+        }
+
+        let total: f64 = stats.iter().map(|(_, v)| v).sum();
+
+        if (total - 0 as f64).abs() < f64::EPSILON {
```
Comment (spenserblack) [inline src/onefetch/language.rs:188]: Is it necessary to subtract by 0? :thinking:

**Review by spenserblack — APPROVED** [review]
LGTM :+1:
Just had 1 question.

File: src/onefetch/language.rs (hunk 1 above)
Comment (o2sh) [reply src/onefetch/language.rs:188]: It looks weird, I aggree; it's a clippy suggestion.

At first, I had written `total == 0 as f64`

Comment (spenserblack) [reply src/onefetch/language.rs:188]: Got it, that makes sense. I assume clippy wouldn't complain if it was this
```suggestion
        if total.abs() < f64::EPSILON {
```

Comment (o2sh) [reply src/onefetch/language.rs:188]: I should really stop relying blindly on clippy suggestions :weary:
Thanks @spenserblack

**Pull request merged**